src/SFA.DAS.FAA.Web/Models/Applications/IndexViewModel.cs

```diff
@@ -0,0 +1,95 @@
+
+        public List<Application> Applications { get; set; } = [];
+        public List<Application> ExpiredApplications { get; set; } = [];
+
+        public class Application
+        {
+            public Guid Id { get; set; }
+            public string VacancyReference { get; set; }
+            public string Title { get; set; }
+            public string EmployerName { get; set; }
+            public string StartedOn { get; set; }
+            public string ClosingDate { get; set; }
+            public bool IsClosingSoon { get; set; }
+            public bool IsClosed { get; set; }
+        }
+
+        public static IndexViewModel Map(ApplicationsTab tab, GetIndexQueryResult source, IDateTimeService dateTimeService)
+        {
+            var result = new IndexViewModel
+            {
+                SelectedTab = tab,
+                PageTitle = $"{tab.GetTabTitle()} applications"
+            };
+
+            foreach (var application in source.Applications.OrderByDescending(x => x.CreatedDate))
+            {
+                var timeUntilClosing = application.ClosingDate.Date - dateTimeService.GetDateTime();
+                var daysToExpiry = (int)Math.Ceiling(timeUntilClosing.TotalDays);
+
+                var closingDate = "";
+                switch (daysToExpiry)
+                {
+                    case < 0:
+                        closingDate = $"Closed on {application.ClosingDate.ToString("dddd d MMMM yyyy", CultureInfo.InvariantCulture)}";
+                        break;
+                    case 0:
+                        closingDate = "Closes today at 11:59pm";
+                        break;
+                    case 1:
+                        closingDate = $"Closes tomorrow ({application.ClosingDate.ToString("dddd d MMMM yyyy", CultureInfo.InvariantCulture)} at 11:59pm)";
+                        break;
+                    case <= 31:
+                        closingDate = $"Closes in {daysToExpiry} days ({application.ClosingDate.ToString("dddd d MMMM yyyy", CultureInfo.InvariantCulture)} at 11:59pm)";
+                        break;
+                    default:
+                        closingDate = $"Closes on {application.ClosingDate.ToString("dddd d MMMM yyyy", CultureInfo.InvariantCulture)} at 11:59pm";
+                        break;
+                }
+
+                var applicationViewModel = new Application
+                {
+                    Id = application.Id,
+                    VacancyReference = application.VacancyReference,
+                    Title = application.Title,
+                    EmployerName = application.EmployerName,
+                    StartedOn =
+                        $"Started on {application.CreatedDate.ToString("d MMMM yyyy", CultureInfo.InvariantCulture)}",
+                    ClosingDate = closingDate,
+                    IsClosingSoon = daysToExpiry is >= 0 and <= 7,
+                    IsClosed = daysToExpiry < 0
```

Comment:
I feel that this should be a Constant somewhere to define what closing soon is, as in X number or days